README.rst

```diff
@@ -43,33 +47,75 @@ The primary use cases are:
 `The documentation <https://jupyter-server-proxy.readthedocs.io/>`_
 contains information on installation & usage.
 
-====================
 Install
-====================
+=======
+
+Python package
+--------------
 
 pip
----
+^^^
 
 .. code-block::
 
    pip install jupyter-server-proxy
 
 conda
------
+^^^^^
 
 .. code-block::
 
    conda install jupyter-server-proxy -c conda-forge
 
-src
----
+JupyterLab extension
+--------------------
 
```

Comment:
Since you're updating this, do you think it's worth mentioning you also need to pip-install the python package too, or is this already clear enough?

Reply:
```suggestion

Note that as the JupyterLab extension only is a graphical interface to launch registered applications in the python package, the extension requires the python package to be installed.

```